README.md

```diff
@@ -223,12 +223,18 @@ Instead, it relies on LVM2 to lock around operations that are not reentrant.
 
 #### Logical volume naming
 
-The volume group name is specified at startup through the
-`-volume-group` argument.
+The volume group name is specified at startup through the `-volume-group` argument.
 
-Logical volumes are named according to the following pattern
-`<volume-group-name>_<logical-volume-name>`.
+Logical volumes names are chosen from randomly-generated, base36-encoded numbers.
```

Comment:
If we use `<prefix>.<fn(CO-specified-name)>` we could use base64-rawurl-encoding here to make the name slightly shorter.

That said, I personally feel that a name with just alphanumerical chars is more readable so base36 looks good.